bitnami/fluentd/values-production.yaml

```diff
@@ -358,6 +371,40 @@ aggregator:
   ##
   # configMap:
 
+  ## Extra configuration to be appended to the fluentd.conf created by default
+  ## ConfigMap created by this chart. For more details on default ConfigMap view
+  ## /templates/aggregator-configmap.yaml
+  ## To learn more about fluend configurations
+  ## https://docs.fluentd.org/configuration/config-file#list-of-directives
+  extraConfig: |
+    # Send all logs to elasticsearch 
+    <match **>
+      @type elasticsearch
+      host elasticsearch-master-headless.default.svc.cluster.local
+      port 9200
```

Comment:
This won't work, right? You need to have an elasticsearch deployed. What if we leave the example in the README but we keep some default values here that doesn't require dependencies with other charts?

Reply:
I didn't get you. Do you mean to remove the complete `match` block from the `values-production.yml` and leave it only as a example in the README.

Reply:
Like mentioned by @andresbono, I would not hardcode this example here as it would add an extra dependency on Elasticsearch, which is not defined in `Chart.yaml` (and thus making this chart impossible to deploy with the default contents of values.yaml).

Reply:
Ok. Got it.